hibernate-core/src/main/java/org/hibernate/SessionFactory.java

```diff
@@ -285,7 +282,7 @@ default void inTransaction(Consumer<Session> action) {
 	 * @deprecated Use the descriptors from {@link #getMetamodel()} instead
 	 */
 	@Deprecated
-	ClassMetadata getClassMetadata(Class entityClass);
+	ClassMetadata getClassMetadata(@SuppressWarnings("rawtypes") Class entityClass);
```

Comment:
Stupid question, why not remove this and other deprecated methods?

Reply:
Sorry, missed that.  Its been integrated, so we should just come behind and make that change